Makefile

```diff
@@ -52,13 +60,30 @@ bindata_assetfs.go: .build/deps-build-go.ok .build/artifacts-bindata_assetfs.$(G
 	go-bindata-assetfs $(GO_BINDATA_FLAGS) -prefix assets -nometadata assets/templates/... assets/static/dist/...
 
 $(NAME): .build/deps-build-go.ok .build/vendor.ok bindata_assetfs.go $(SOURCES)
-	go build -ldflags "-X main.version=$(VERSION)"
+	$(GO) build -ldflags "-X main.version=$(VERSION)"
 
 .build/vendor.ok: .build/deps-build-go.ok Gopkg.lock Gopkg.toml
 	dep ensure
 	dep prune
 	touch $@
 
+build: promu
+	@echo ">> building binaries"
```

Comment:
There's no `@echo` usage for any other target, no need to make any new target special in that regard

Reply:
Again, those are not practices established here, nor is there any reason to comply here